ci-operator/config/smongiar/xpaas-qe/smongiar-xpaas-qe-7.11.x__smongiar-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,40 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  test-image:
+    name: xpaas-qe
+    namespace: ci
+    tag: 7.11.x-openjdk1.8_local
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
```

```suggestion
      version: "4.14"
```
With the GA of 4.13 happening this week, we now need to transition all existing work to 4.14 as we are meant to be testing pre-release OCP versions.

The filename of this config also needs to be updated from 4.13 -> 4.14